examples/flutter_gallery/lib/demo/material/backdrop_demo.dart

```diff
@@ -301,9 +301,9 @@ class _BackdropDemoState extends State<BackdropDemo> with SingleTickerProviderSt
 
     final double flingVelocity = details.velocity.pixelsPerSecond.dy / _backdropHeight;
     if (flingVelocity < 0.0)
-      _controller.fling(velocity: math.max(2.0, -flingVelocity));
+      _controller.fling(velocity: math.max(2, -flingVelocity));
     else if (flingVelocity > 0.0)
```

Comment:
And here?

Reply:
There is absolutely no way we can convert 2.0 here to an integer, not without explicitly writing the `<double>` type argument. There is no double context here except the one inferred from the types of the argument, so we can't use the context to decide the types of the arguments.
If we just make it an integer, then the inferred type argument of `math.max` becomes `<num>`, and the result might be the integer 2.

Reply:
I should leave the reviews to the experts 😄  So is this a bug in `dartfix`?

I think this illustrates the issue:

```
  final a = math.max(2.0, 1.5); // analysis reports a as 'double'
  final b = math.max(2, 1.5); // analysis reports b as 'num'

  double a2 = a;
  double b2 = b; // runtime error: 'int' is not a subtype of type 'double'
```